neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -34,8 +36,84 @@ def GetWallet1(cls, recreate=False):
                                            to_aes_key(UserWalletTestCase.wallet_1_pass()))
         return cls._wallet1
 
+    @classmethod
+    def OpenWallet(cls):
+        PromptData.Wallet = cls.GetWallet1(recreate=True)
+
+    @classmethod
+    def tearDown(cls):
+        PromptData.Wallet = None
+
     # Beginning with refactored tests
 
+    def test_wallet(self):
+        with patch('neo.Prompt.PromptData.PromptData.Prompt'):
+            res = CommandWallet().execute(None)
+            self.assertEqual(res, None)
+
+    def test_wallet_wrong_command(self):
+        with patch('neo.Prompt.PromptData.PromptData.Prompt'):
```

Comment:
this does not test invalid command. It passes because it has no wallet open and therefore returns `None`.

Reply:
if your IDE doesn't support coverage highlighting you could alternatively run `make coverage` or have a look at the `coveralls` output in your PR as a last resort.